mslib/msui/performance_settings.py

```diff
@@ -30,13 +30,13 @@
 from PyQt5 import QtCore, QtWidgets
 
 from mslib.utils import FatalUserError
-from mslib.msui import aircrafts, constants
+from mslib.msui import aircraft, constants
```

Comment:
that's an API change. The issue the work is based on was added for 10.0.0.

To get the valuable other changes in could you undo the API renamings and move this work to an issue e.g. improve names for the next major?

Reply:
> The name maybe was also choosen for adding more than only the SimpleAircraft class.

The plural of "aircraft" is "aircraft", not "aircrafts". There is nothing wrong with adding more Aircraft classes in there.

> that's an API change. The issue the work is based on was added for 10.0.0.

Is the modules name part of an external API? How is it used from outside of the MSS codebase?

Reply:
that's a good question. I don't know how people use our modules.

Semantic versioning is all about communicating the nature and scope of the changes to the code through the changes to the version number.

I don't want to do that differently.

Reply:
> The plural of "aircraft" is "aircraft", not "aircrafts". There is nothing wrong with adding more Aircraft classes in there.

interesting, it is defining the family of helicopter, balloon, airplane ...

Reply:
We don't warn users writing plugins about parts we still see as unstable. That module was introduced 8 year ago. It rather won't have that category. 
If someone wants to add to an output plugin performance data it can be done.

Usually users ask only if they need help. So it is difficult to see what they have done.

Reply:
> Semantic versioning is all about communicating the nature and scope of the changes to the code through the changes to the version number.

Semantic versioning is about communicating the nature and scope of the changes to the _public API of the code_ through the changes to the version number. What happens internally doesn't matter, unless it affects the behavior of the public API.

Unfortunately, python is ill-equipped to make this distinction (other than the leading underscore convention), so one could argue that every detail is automatically part of the public API.

Anyway, I can move that particular change to a different PR to develop.